modules/govuk/manifests/node/s_draft_frontend.pp

```diff
@@ -14,6 +14,7 @@
     'PLEK_HOSTNAME_PREFIX': value => 'draft-';
     'PLEK_SERVICE_ERRBIT_URI': value => "https://errbit.${app_domain}";
     'PLEK_SERVICE_IMMINENCE_URI': value => "https://imminence.${app_domain}";
+    'PLEK_SERVICE_LICENSIFY_URI': value => "https://licensify-admin.${app_domain}";
```

Comment:
The environment variable name is constructed by plek, so if you say `Plek.find('cats')`, it will check PLEK_SERVICE_CATS_URI [1]. Looking at frontend, it asks Plek to find 'licensify' so I think that is right for the environment variable name.

One think I looked in to was the `licensify-admin` bit. Given that this was not specified before, on the live stack, asking Plek for licensify would have probably been falling through to `licensify.publishing.service.gov.uk`. I checked govuk-puppet, and it looks like licensify and licensify-admin are both valid hosts for the same service, so I think that is ok also.

1: https://github.com/alphagov/plek/blob/master/lib/plek.rb#L168